server/repository/src/migrations/v2_07_04/create_sync_message_table.rs

```diff
@@ -5,34 +5,35 @@ pub(crate) struct Migrate;
 
 impl MigrationFragment for Migrate {
     fn identifier(&self) -> &'static str {
-        "create_message_table"
+        "create_sync_message_table"
```

Comment:
Changing identifier would mean migration will re-run (in case devs testing this already ran the previous migration). Don't need to delete possible message table